ledger/complete/wal/compactor_test.go

```diff
@@ -113,7 +113,7 @@ func Test_Compactor(t *testing.T) {
 			select {
 			case <-co.done:
 				// continue
-			case <-time.After(20 * time.Second):
+			case <-time.After(70 * time.Second):
```

Comment:
this might be a bit high,

Reply:
I tried with less, and the test wouldn't pass..  I'm open to suggestions.